source/draw/gpu/opengl/backend/EglDisplayContext.ooc

```diff
@@ -0,0 +1,137 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+version(!gpuOff) {
+use base
+use geometry
+import egl/egl
```

Comment:
yeah and as my next comment shows, maybe "generally outside in other files" isn't even true